__tests__/app.test.js

```diff
@@ -580,4 +580,32 @@ describe("GET /api/articles ADDITIONAL FEATURE - Filer by topic", () => {
         );
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles/:articleid ADDITIONAL FEATURE - Filer by topic", () => {
+  test("should return 200", () => {
+    return request(app).get("/api/articles/1").expect(200);
```

Comment:
This test is a duplication of what is being done in the one below, so no requirement for it here 🙂